index.html

```diff
@@ -1,140 +1,170 @@
-							<h3 id="bash" >Git BASH</h3>
-							<p>Git for Windows provides a BASH emulation used to run Git from the command line.  *NIX users should feel right at home, as the BASH emulation behaves just like the "git" command in LINUX and UNIX environments.</p>
-							<h3 id="git-gui" >Git GUI</h3>
-							<p>As Windows users commonly expect graphical user interfaces, Git for Windows also provides the Git GUI, a powerful alternative to Git BASH, offering a graphical version of just about every Git command line function, as well as comprehensive visual diff tools.</p>
-							<h3 id="explorer" >Shell Integration</h3>
-							<p>Simply right-click on a folder in Windows Explorer to access the BASH or GUI.</p>
-						</div>
-					</div>
-					<div class="vcentercontainer detailsimg">
-						<div class="verticalgallery">
-							<a class="fancybox-thumb" data-fancybox-group="gallery" href="img/gw1.png" title="Git BASH">
-								<img src="img/gw1web_thumb.png" alt="Git BASH">
-							</a>
-							<a class="fancybox-thumb" data-fancybox-group="gallery" href="img/gw2.png" title="Git GUI">
-								<img src="img/gw2web_thumb.png" alt="Git GUI">
-							</a>
-							<a class="fancybox-thumb" data-fancybox-group="gallery" href="img/gw3.png" title="Gitk: Exploring Commits">
-								<img src="img/gw3web_thumb.png" alt="Gitk: Exploring Commits">
+						</p>
+						<p>
+							<a name="download" /><a class="btn btn-primary shadow mr-2"
+								href="https://github.com/git-for-windows/git/releases/latest"
+								target="_blank">Download</a>
+							<a class="btn btn-outline-primary shadow mr-2" href="#contribute">Contribute</a>
+						</p>
+						<p class="h2" id="version">
+							<a href="https://github.com/git-for-windows/git/releases/tag/v2.34.0.windows.1" title="Version 2.34.0 was published on Mon, 15 Nov 2021 19:52:38 UTC">
+								Version 2.34.0
 							</a>
-						</div>
+						</p>
 					</div>
 				</div>
-				<div class="stud"></div>
-				<h2 id="contribute">Contribute</h2>
-				<div class="content contributecontent">
-					<div class="vcentercontainer detailsimg">
-						<div>
-							<img src="img/details_features.png" alt="BASH Drawing">
-							<img src="img/details_contribute.png" alt="Git for Windows Logo">
-						</div>
+			</div>
+		</div>
+	</section>
+	<section>
+		<div class="container">
+			<h2>Tools &amp; Features</h2>
+			<p class="text-muted mb-5">Git for Windows focuses on offering a lightweight, native set of tools that bring
+				the full feature set of the <a href="https://git-scm.com" target="_blank">Git SCM</a> to Windows while
+				providing appropriate user interfaces for experienced Git users and novices alike.</p>
+			<div class="row">
+				<div class="col-sm-6 col-lg-4 mb-3">
+					<img src="img/gw1web_thumb.png" alt="Git BASH" class="img-other-logos">
+					<h3>Git BASH</h3>
```

Comment:
The all-caps "BASH" here stands out, and looks odd. Although it stands for Bourn Again SHell, it's normally stylised as "Bash": https://www.gnu.org/software/bash/

Reply:
This was copied over but makes sense, will change.

<img width="565" alt="Screen Shot 2021-11-18 at 8 20 11 PM" src="https://user-images.githubusercontent.com/12194950/142435460-6e5902ce-cd54-4589-bc1b-4df62896b6bf.png">